### ISSUE TITLE
Missing Flush File can lead to data loss

### ISSUE THREAD

**Issue by danzu82** [fieldrndservices/logger#11]
We're investigating the Logger framework for use on a real-time system. It seems to work fine so far, except one issue. When the controller shuts down unexpectedly (power loss in the middle of operation) the logfile is deleted after the restart. It might be because the file is never actually written to disk, but is only in memory.
I looked at the code, and I added a Flush File in the False case in File.lvlib:Write.vi. This seems to resolve the issue, the logfile is still there after an unexpected shutdown.
I'm aware this might have consequences on the performance, since this forces the file write to disk to be synchronous instead of leaving the decision when to actually write to file to the OS. So probably the best thing would be to make this configurable.

My workaround:
![image](https://github.com/fieldrndservices/logger/assets/148533466/45f2d17d-28d1-41ef-8fb7-7435b315a971)

Other minor points:

- File.lvlib:Manager.vi and some other VIs have debugging enabled. This is not ideal for performance reasons.
- Did you consider setting the "Separate compiled code from source file" checkbox?

**Comment by volks73** [comment]
I agree that it should be a configuration. I would happy to accept and merge a Pull Request (PR) implementing this feature and creating a new release.
